### ISSUE TITLE
Where do models get downloaded?

### ISSUE THREAD

**Issue by LitenBuzzTh** [facebookresearch/audiocraft#106]
As I started using audiocraft, I noticed that it downloads the models in Drive C by default and this makes way to al lot of problems.
As other UIs like automatic1111 or oobabooga, it is common to have a folder named "models" inside the main repo directory, and not some random place in drive C

Also it would be nice to be able to download models manually from HF and putting the in the supposed "models" folder and it detects them without needing to download them again.

thanks

**Comment by sdbds** [comment]
https://github.com/sdbds/audiocraft-webui
this is one-click install version and it fixed that
1 windows show error for triton
2 download model in local dir with it 'huggingface' in you put dir

**Comment by LitenBuzzTh** [comment]
> https://github.com/sdbds/audiocraft-webui this is one-click install version and it fixed that 1 windows show error for triton 2 download model in local dir with it 'huggingface' in you put dir


Is it possible to manually download model weights from HuggingFace.com and put them in "huggingface" directory?

**Comment by sdbds** [comment]
> > https://github.com/sdbds/audiocraft-webui this is one-click install version and it fixed that 1 windows show error for triton 2 download model in local dir with it 'huggingface' in you put dir
> 
> Is it possible to manually download model weights from HuggingFace.com and put them in "huggingface" directory?

need change code and package model to other format

**Comment by Magenta-6** [comment]
Models get downloaded to:
C:\Users\XYZ\.cache\huggingface\hub\models--facebook--musicgen-melody\blobs

**Comment by Wickedesigns** [comment]
> Models get downloaded to: C:\Users\XYZ.cache\huggingface\hub\models--facebook--musicgen-melody\blobs

Right. And my directory is different, I searched "models--facebook--musicgen-melody" in C disk, and all models showed up after a while...

**Comment by Wickedesigns** [comment]
> Models get downloaded to: C:\Users\XYZ.cache\huggingface\hub\models--facebook--musicgen-melody\blobs

Right. And my directory is different, I searched "models--facebook--musicgen-melody" in C disk, then the models showed up...

**Comment by Wickedesigns** [comment]
> Models get downloaded to: C:\Users\XYZ.cache\huggingface\hub\models--facebook--musicgen-melody\blobs

Right. And my directory is different, I searched "models--facebook--musicgen-melody" in C disk, later the models showed up...
